ec2-cluster/modules/node/templates/setup

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+set -e
+
+mkdir -p /tmp/tessellation/dag
+mkdir -p /tmp/tessellation/data/snapshot
+
+wget https://github.com/Constellation-Labs/tessellation/releases/download/v${tessellation_version}/cl-dag-l1.jar -q -O /tmp/tessellation/cl-dag-l1.jar
+wget https://github.com/Constellation-Labs/tessellation/releases/download/v${tessellation_version}/cl-node.jar -q -O /tmp/tessellation/cl-node.jar
+wget https://github.com/Constellation-Labs/tessellation/releases/download/v${tessellation_version}/cl-keytool.jar -q -O /tmp/tessellation/cl-keytool.jar
+# wget https://constellationlabs-dag.s3.amazonaws.com/keys_v2/${key} -q -O /tmp/tessellation/key.p12
+wget https://constellationlabs-dag.s3.amazonaws.com/keys_v2/data.csv -q -O /tmp/tessellation/genesis.csv
+wget https://constellationlabs-dag.s3.us-west-1.amazonaws.com/${app_env}-seedlist -q -O /tmp/tessellation/${app_env}-seedlist
+
+cp /tmp/rollback /tmp/tessellation/rollback
+cp /tmp/genesis /tmp/tessellation/genesis
+cp /tmp/validator /tmp/tessellation/validator
+cp /tmp/join /tmp/tessellation/join
+cp /tmp/l1-join /tmp/tessellation/dag/join
+cp /tmp/update-seedlist /tmp/tessellation/update-seedlist
+cp /tmp/update-version /tmp/tessellation/update-version
+cp /tmp/l1-initial-validator /tmp/tessellation/dag/initial-validator
+cp /tmp/l1-validator /tmp/tessellation/dag/validator
+cp /tmp/restart /tmp/tessellation/restart
+cp /tmp/l1-restart /tmp/tessellation/dag/restart
+cp /tmp/snapshots-s3-sync /tmp/tessellation/snapshots-s3-sync
+cp /tmp/key.p12 /tmp/tessellation/key.p12
+
+sudo mv /tmp/tessellation /home/admin
+sudo chmod u+x /home/admin/tessellation/rollback
```

Comment:
Maybe paths `/home/admin/tessellation/l0` and `/home/admin/tessellation/l1`? 

Currently, L1 is installed in the subdirectory of L0 like it's a subresource which it isn't